src/components/Tables/Table.tsx

```diff
@@ -149,13 +149,15 @@ interface Props<Data, DbData extends Record<string, any>, PaginationType> {
             ? ClientSideFilter<Data, any>
             : ServerSideFilter<Data, any, DbData>
     >;
+    parcelRowBreakPoints?: number[]; //This is a list of indices where the line will be drawn when sorting by packing date/slot
     defaultShownHeaders?: readonly (keyof Data)[];
     toggleableHeaders?: readonly (keyof Data)[];
     columnDisplayFunctions?: ColumnDisplayFunctions<Data>;
     columnStyleOptions?: ColumnStyles<Data>;
     editableConfig: EditableConfig<Data>;
     onRowClick?: OnRowClickFunction<Data>;
     pointerOnHover?: boolean;
+    setSortedColumn?: (sortedColumn: string) => void;
```

Comment:
we should be able to remove this additional prop if we use the sortState to extract the sorted column. If for whatever reason we do need it, please move this prop into the sortConfig.